.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,10 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.202.5/containers/python-3
 {
 	"name": "Python 3",
-	"runArgs": ["--init"],
+	"runArgs": [
+        "--init",
+        "--network=host" // Needed for using localhost within devcontainer
+    ],
```

Comment:
There seems to be a mix of tabs and spaces throughout `devcontainer.json`. I'm not sure what the standard is, but whichever is the correct one should be consistently used.